internal/pkg/cli/pipeline_init.go

```diff
@@ -683,15 +689,18 @@ func (o *initPipelineOpts) createBuildspec() error {
 		buildspecExists = true
 		buildspecPath = e.FileName
 	}
-	buildspecMsgFmt := "Wrote the buildspec for the pipeline's build stage at '%s'\n"
-	if buildspecExists {
-		buildspecMsgFmt = "Buildspec file for pipeline already exists at %s, skipping writing it.\n"
-	}
 	buildspecPath, err = relPath(buildspecPath)
 	if err != nil {
 		return err
 	}
-	log.Successf(buildspecMsgFmt, color.HighlightResource(buildspecPath))
+	buildspecMsgFmt := "Wrote the buildspec for the pipeline's build stage at '%s'\n"
+	if buildspecExists {
+		buildspecMsgFmt = `Buildspec file for pipeline already exists at %s, skipping writing it.
+Previously set config will remain.`
```

Comment:
I think we might need a new line at the end of this string
```suggestion
Previously set config will remain.
`
```

Reply:
Good catch. Thanks!